app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -233,8 +233,8 @@ const getAnnualIncome = ({ monthlyIncome, annualIncome }) => {
 
 const setApplicationsDefaults = (application) => {
   const applicationWithDefaults = cloneDeep(application)
-
   applicationWithDefaults.annual_income = getAnnualIncome({monthlyIncome: application.monthly_income, annualIncome: application.annual_income})
+  applicationWithDefaults.lease.no_preference_used = !application.lease.preference_used
```

Comment:
I initially moved this to the `LeaseInformationInputs` component but kept it here so it stays centralized to this function.